analyses/mutational-signatures/.gitignore

```diff
@@ -1,2 +1,8 @@
 # ignore folders with plots for individual biospecimens
 individual_mutation_sig
+
+# Ignore large file from de novo signature extraction
+results/denovo_sigfit_signatures.RDS
+
+# Extraneous plot output
+Rplots.pdf
```

Comment:
Where does this plot come from?

Reply:
When running `sigfit::extract_signatures()` from the command line, you'll get the goodness-of-fit plot as `Rplots.pdf` in the directory you call the script from. When you give a range of integers for `nsignatures`, `sigfit::extract_signatures()` runs `sigfit::plot_gof()` but you don't have control over where to save this plot in your arguments to `extract_signatures()` as far as I can tell. So that is why I've implemented things this way more generally.